src/lib/webpack/webpack-client-config.js

```diff
@@ -52,15 +55,15 @@ function clientConfig(env) {
 					loader: resolve(__dirname, './async-component-loader'),
 					options: {
 						name(filename) {
-							let relative = filename.replace(src, '');
-							let isRoute = filename.indexOf('/routes/') >= 0;
-
-							return isRoute ? 'route-' + relative.replace(/(^\/(routes|components\/(routes|async))\/|(\/index)?\.js$)/g, '') : false;
+							filename = normalizePath(filename);
+							if (!filename.includes('/routes/')) return false;
+							let relative = filename.replace(normalizePath(src), '');
+							return 'route-' + cleanFilename(relative);
 						},
 						formatName(filename) {
-							let relative = filename.replace(source('.'), '');
-							// strip out context dir & any file/ext suffix
-							return relative.replace(/(^\/(routes|components\/(routes|async))\/|(\/index)?\.js$)/g, '');
+							filename = normalizePath(filename);
+							let relative = filename.replace(normalizePath(source('.')), '');
```

Comment:
Any specific reason to use `source('.')` here instead of `src` like above?

Reply:
maybe noone ever changed it, not sure